src/ranged.cpp

```diff
@@ -968,13 +990,25 @@ static int print_ranged_chance( const player &p, const catacurses::window &w, in
     }
 
     if( display_type != "numbers" ) {
-        std::string symbols;
+        int column_number = 1;
+        if( !( panel_type == "compact" || panel_type == "labels-narrow" ) ) {
+            auto label = _( "Symbols:" );
```

Comment:
Please get rid of `auto`s here and everywhere where it makes sense and replace them with exact types.